compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4576,7 +4576,7 @@ TR::Register *OMR::Power::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::
    bool  supportsVSX = (processor >= TR_PPCp8) && !disableVSXArrayCopy && TR::Compiler->target.cpu.getPPCSupportsVSX();
 
    static bool disableLEArrayCopyHelper  = (feGetEnv("TR_disableLEArrayCopyHelper") != NULL);
-   static bool disableVSXArrayCopyInlining = (feGetEnv("TR_disableVSXArrayCopyInlining") != NULL);
+   static bool disableVSXArrayCopyInlining = (feGetEnv("TR_disableVSXArrayCopyInlining") == NULL); // Disabling due to a performance regression
```

Comment:
Is it better to rename this env var ? As it stands, the env var is named TR_disableVSXArrayCopyInlining but if someone sets it, then it enables the arraycopy inlining (which is a bit odd).